views/canvas.ejs

```diff
@@ -114,7 +114,13 @@
 
     $('#playbutton').click(function(){
     var clicked = $(this).data('clicked');
-
+      
+    var volume_s = document.getElementById('volume_s');
+    volume_s.onchange = function(){
+      console.log(this.value)
```

Comment:
No need to display on console though.

Reply:
![image](https://user-images.githubusercontent.com/14250095/39417598-9649cfe0-4c7f-11e8-9643-d62dc8f2beb5.png)